### PR TITLE
Add FormattedDate spec

### PR DESCRIPTION
**Pull request by danieledamiani**

### CODE REVIEW

File: packages/tempus-client_v3/src/components/shared/FormattedDate/FormattedDate.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ const FormattedDate: FC<FormattedDateProps> = props => {
   }
 
   return (
-    <span className="tc__formatted-date">
+    <span className="tc__formatted-date" data-testid="formatted-date">
```
Comment (lazarmitic) [inline packages/tempus-client_v3/src/components/shared/FormattedDate/FormattedDate.tsx:56]: Do we need this testid attr?

Comment (ShawTim) [reply packages/tempus-client_v3/src/components/shared/FormattedDate/FormattedDate.tsx:56]: i think we can do:
```
const { container } = subject(defaultProps);
container.querySelector('xxx');
```
not a fan to add `data-testid` becoz of unit test

Comment (danieledamiani) [reply packages/tempus-client_v3/src/components/shared/FormattedDate/FormattedDate.tsx:56]: to target the parent container, if there's a better solution it's more than welcome :)

Comment (danieledamiani) [reply packages/tempus-client_v3/src/components/shared/FormattedDate/FormattedDate.tsx]: fixed

**Review by ShawTim — APPROVED** [review]

**Pull request merged**